.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install dependencies
+        run: |
+          python -m pip install -U pip
+          python -m pip install -U docutils
+
+      - name: Build
+        run: make -j$(nproc)
+
+      - name: Deploy
+        if: >
+          (
+            github.repository == 'python/peps' &&
+            github.ref == 'refs/heads/master'
+          )
+
+        run: |
+          bash deploy.bash
```

Comment:
Is this actually triggering a deployment?

Reply:
Yes, but only for the `python/peps` repo and `master` branch (like [`.travis.yml`](https://github.com/python/peps/blob/5f9a50d3d43f3b2b921ade84c3d6ca4dbf248480/.travis.yml#L19-L20)).

As a demo, here's another branch that allows a deploy for my fork and that branch:

* https://github.com/hugovk/peps/compare/lint-rsk-backticks...hugovk:TEMP-lint-rsk-backticks---deploy-from-fork

The deploy is triggered (and fails, as expected due to lack of credentials in my repo):

* https://github.com/hugovk/peps/runs/1285567375?check_suite_focus=true